css/index.css

```diff
@@ -104,14 +132,48 @@ p {
 .middle-content .boxes .box {
     width: 12.5%;
     height: 100px;
-    background: black;
+    /* background: black; */
     margin: 20px 2.5%;
     color: white;
     display: flex;
     align-items: center;
     justify-content: center;
 }
 
```

Comment:
Great job using parent/child relationships to inherit the style from the .middle-content .boxes .box class.  You only changed the color in the individual boxes.